EcosiaTests/SnapshotTests/Onboarding/OnboardingTests.swift

```diff
@@ -0,0 +1,35 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SnapshotTesting
+import XCTest
+@testable import Client
+
+final class OnboardingTests: SnapshotBaseTests {
+    
+    func testWelcomeScreen() {
+        SnapshotTestHelper.assertSnapshot(initializingWith: {
+            Welcome(delegate: MockWelcomeDelegate())
+        }, locales: LocalizationOverrideTestingBundle.supportedLocales, wait: 1.0)
+    }
+    
+    func testWelcomeStepsScreens() {
+        // Number of steps in the WelcomeTour
+        let numberOfSteps = 4
+        // Iterate through steps and take snapshots, skipping the first one
+        for step in 1...numberOfSteps {
+            let startingStep = WelcomeTour.Step.all[step-1]
+            // Precision at .95 to accommodate a snapshot looking slightly different due to the different data output
+            // from the statistics json
```

Comment:
Did you look into mocking the json? Not major but might be interesting to test how different statistics look as well

Reply:
Yes we'd mock it but that would require further work. Would like to tackle it separately